cosmos-sdk-proto/src/lib.rs

```diff
@@ -7,7 +7,7 @@
 
 #![warn(trivial_casts, trivial_numeric_casts, unused_import_braces)]
 #![forbid(unsafe_code)]
-//#![doc(html_root_url = "https://docs.rs/cosmos-sdk-proto/0.4.0")]
+#![doc(html_root_url = "https://docs.rs/cosmos-sdk-proto/0.2.0")]
```

Comment:
Seems someone wasn't wild about this. There are a few different ways to do it including putting it in `Cargo.toml` (and potentially omitting a version).

This approach ensures every crate release links to the documentation for that release and makes it available to the widest variety of tooling, but if it seems too annoying we can consider other options.